hero/examples/core-server.mjs

```diff
@@ -1,11 +1,11 @@
-import Core from '@secret-agent/core';
+import Core from '@ulixee/hero-core';
 
 (async () => {
   Core.onShutdown = () => {
     console.log('Exiting Core Process');
     process.exit();
   };
-  await Core.start({ coreServerPort: 1337 }, !process.env.SA_TEMPORARY_CORE);
+  await Core.start({ coreServerPort: 1337 }, !process.env.ULX_TEMPORARY_CORE);
```

Comment:
I know there's a TON of stuff here, but I'm finding the times we're using ULX vs Hero to be ambiguous

Reply:
I believe it's just for env vars, which I agree can just as well be HERO_.

Reply:
Updated